Samples~/WebGLExample/Editor/EditorWindows/SetupTemplate/SetupWebGLTemplate.uxml

```diff
@@ -0,0 +1,15 @@
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" xsi="http://www.w3.org/2001/XMLSchema-instance" engine="UnityEngine.UIElements" editor="UnityEditor.UIElements" noNamespaceSchemaLocation="../../UIElementsSchema/UIElements.xsd" editor-extension-mode="True">
+    <ui:VisualElement name="PartnerSettingsHeading" class="heading" />
+    <ui:VisualElement name="AvatarSettings" style="flex-grow: 0; background-color: rgba(0, 0, 0, 0); height: auto; align-self: auto; justify-content: flex-start; align-items: stretch; flex-direction: flex-start; flex-shrink: 0;">
+        <ui:VisualElement name="AvatarSettingsHeading" class="heading" />
+    </ui:VisualElement>
+    <ui:VisualElement name="AvatarCaching" style="flex-grow: 0; background-color: rgba(0, 0, 0, 0); flex-shrink: 0; margin-left: 8px; margin-right: 8px; margin-top: 8px; margin-bottom: 8px;">
+        <ui:VisualElement name="AvatarCachingHeading" class="heading">
+            <ui:Label text="To get it working, move the &quot;WebGLTemplate&quot; folder to the main directory and select &quot;RPMTemplate&quot; as the template in the player settings. You can make it happen automatically by clicking &quot;OK.&quot;" display-tooltip-when-elided="true" class="headingText" style="white-space: normal;" />
```

Comment:
I think the wording of the text in the window can be improved. 
Something more like
"This example includes a WebGL template that can be used for WebGL builds. To use the template it needs to be moved inside WebGLTemplates folder. Would you like to move it automatically? "

Again maybe something to check with Bernhard, he might have some better ideas.

Reply:
Yeah, was thinking about this as well. Will investigate